tools/ci/tcdownload.py

```diff
@@ -71,7 +71,7 @@ def run(*args, **kwargs):
 
     if not taskgroups:
         logger.error("No complete TaskCluster runs found for ref %s" % kwargs["ref"])
-        return
+        return 1
```

Comment:
Isn't `sys.exit(run(None, vars(kwargs)))` or similar needed below for this to work?

Reply:
The result of the  `run()` function for `tc-download` is captured by `wpt` in https://github.com/web-platform-tests/wpt/blob/5336ed0a7fa94b3ecd1775a495acf55c71e6c022/tools/wpt/wpt.py#L166)

Actually, the `tc-download` is a subcommand of `wpt`. Example:

```
./wpt tc-download --ref a7f3642cf9af7321211ac05cd06e0512fb4d6372  --repo-name wpt --user user --password XXXXXXXX
```

Reply:
BTW @clopez has an interesting extension for this which we are using to get results from task-cluster and upload it the wpt.fyi repository: https://github.com/psaavedra/wpt/commit/c093e312aec77f276d07c7dc8423ca26879824fe

Reply:
Thanks, then I understand the context. Writing a longish explanation in the commit message is OK and helpful in review.